### PR TITLE
use multiple ES hosts if available

### PR DESCRIPTION
**Pull request by snopoke**
This should make doing rolling restart without downtime possible.

Together with: https://github.com/dimagi/commcarehq-ansible/pull/332

See also https://github.com/dimagi/commcarehq-ansible/pull/331

@TylerSheffels 
buddy @biyeun

### CODE REVIEW

**Comment by benrudolph** [comment]
sweet this is awesome

File: corehq/elastic.py [hunk 1]
```diff
@@ -18,15 +18,23 @@
...
     Get a handle to the configured elastic search DB.
     Returns an elasticsearch.Elasticsearch instance.
     """
-    return Elasticsearch([{
-        'host': settings.ELASTICSEARCH_HOST,
-        'port': settings.ELASTICSEARCH_PORT,
-    }])
+    es_hosts = getattr(settings, 'ELASTICSEARCH_HOSTS', None)
+    if not es_hosts:
+        es_hosts = [settings.ELASTICSEARCH_HOST]
```
Comment (esoergel) [inline corehq/elastic.py]: `#ytilibitapmoc`

**Comment by esoergel** [comment]
:100:

File: corehq/elastic.py (hunk 1 above)
Comment (snopoke) [inline corehq/elastic.py]: ?

Comment (benrudolph) [inline corehq/elastic.py]: look at it in a mirror!

Comment (snopoke) [inline corehq/elastic.py]: :tongue: :cactus:

**Pull request merged**